### PR TITLE
build: remove software-properties-common and set OpenCL 20 on CI

### PR DESCRIPTION
**Pull request by ilya-naumov**

### CODE REVIEW

File: templates/ubuntu20/hw/gpu.dockerfile.j2 [hunk 1]
```diff
@@ -15,6 +15,9 @@ RUN apt-get update && \
     rm -rf /var/lib/apt/lists/*
 {% else %}
 WORKDIR ${INTEL_OPENVINO_DIR}/install_dependencies
-RUN ./install_NEO_OCL_driver.sh --no_numa -y -d ${INTEL_OPENCL} && \
+RUN apt-get update && \
+    apt-get install -y --no-install-recommends software-properties-common && \
```
Comment (generalova-kate) [inline templates/ubuntu20/hw/gpu.dockerfile.j2:19]: I saw that Artym added installation such package in the script, we need only remove it

Comment (ilya-naumov) [reply templates/ubuntu20/hw/gpu.dockerfile.j2]: Thanks!

Comment (generalova-kate) [reply templates/ubuntu20/hw/gpu.dockerfile.j2]: how about rhel images?

Comment (ilya-naumov) [reply templates/ubuntu20/hw/gpu.dockerfile.j2]: Changes for RHEL are not required

**Pull request merged**